Documentation/Index.rst

```diff
@@ -57,7 +57,7 @@ LDAP / SSO Authentication
 	**Other version of this document**
 
 	:PDF:
-		https://docs.typo3.org/typo3cms/extensions/ig_ldap_sso_auth/3.3.0/_pdf/
+		https://docs.typo3.org/typo3cms/extensions/ig_ldap_sso_auth/3.3.1/_pdf/
```

Comment:
This is not related to the proposed change

Reply:
Sorry, that one was not intended